core/config.lua

```diff
@@ -67,6 +67,12 @@ TC2.defaultConfig.bar = {
 	colorMod			= 0,									-- color modifier
 }
 
+-- self threat bar color settings
+TC2.defaultConfig.selfBar = {
+	enableCustomColor 		 = false,							-- enable custom color for player
+	customColor       		 = {0.58, 0, 0.827, 0.35},			-- custom color for player
```

Comment:
and this just to `color`

Reply:
also why was this value chosen as a default?

I would go with `{0.8, 0, 0, 1}` (a not fully saturated red)

Reply:
Renamed.
When I tested I though red color could be used for something (green one used for pets), so I went with purple (demon hunter style :) )